### PR TITLE
Update .env.template

### PR DESCRIPTION
**Pull request by Gustavsenay**
Replaced all hyphens (-) with underscores (_) in the parameter placeholders that the user needs to set, making it easier to select and paste the correct keys by simply double-clicking them

### Change Safety

- [ ] I have added tests to cover my changes
- [x] I have considered potential risks and mitigations for my changes

### CODE REVIEW

**Comment by nponeccop** [comment]
@Torantulino Close as not necessary? The double clicking is entirely editor-specific.

As a counterexample, the clicking on the dashed placeholders works just fine in vim :)

**Comment by Gustavsenay** [comment]
@nponeccop is just an UX improvement, but I get your point, maybe is better to straight leave those empty? I'm also thinking that an install.bat would be more user friendly to set all the keys, any thoughts about that?

**Comment by nponeccop** [comment]
@Torantulino has the last saying, so let's wait for him to have time to look

**Comment by aosan** [comment]
@Gustavsenay love the split of main.py into  role based modules, there are way too many exceptions to handle in a single place. We have multiple back-ends, probably more coming. I've seen the PRs for web front-end. If Pinecode breaks, people can use Redis and vice-versa. 

Empty values in .env is better UX and validation handling. Ideally, modules should check for critical configs in .env, exit with a helpful if desired values are missing. In this way, every future module, integration etc. can do its own validation of configuration with helpful exceptions handling.

@Torantulino this PR is worth considering, would take care of dozens of configuration issues already reported.

**Comment by nponeccop** [comment]
It shouldn't be called "Update .env.template". @Gustavsenay can you provide a better title?

File: .env.template [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-OPENAI_API_KEY=your-openai-api-key
-ELEVENLABS_API_KEY=your-elevenlabs-api-key
-SMART_LLM_MODEL="gpt-4"
-FAST_LLM_MODEL="gpt-3.5-turbo"
+PINECONE_API_KEY=
+PINECONE_ENV=
+OPENAI_API_KEY=
+ELEVENLABS_API_KEY=
+SMART_LLM_MODEL="gpt_4"
+FAST_LLM_MODEL="gpt_3.5_turbo"
```
Comment (nponeccop) [inline .env.template:6]: What is this? The dashes are in the model names from the API, so changing them to underscores is counterproductive.

File: .env.template [hunk 2]
```diff
@@ -1,14 +1,14 @@
-PINECONE_API_KEY=your-pinecone-api-key
-PINECONE_ENV=your-pinecone-region
-OPENAI_API_KEY=your-openai-api-key
-ELEVENLABS_API_KEY=your-elevenlabs-api-key
-SMART_LLM_MODEL="gpt-4"
-FAST_LLM_MODEL="gpt-3.5-turbo"
+PINECONE_API_KEY=
+PINECONE_ENV=
+OPENAI_API_KEY=
```
Comment (nponeccop) [inline .env.template:3]: The previous version was self-documenting, making it easier for people to fill. So I guess the premise of this PR is just wrong. @Torantulino ?

File: .gitignore
Comment (nponeccop) [inline .gitignore:1]: Unrelated to other changes. And this is already in #611 . Remove

File: scripts/ai_config.py
Comment (nponeccop) [inline scripts/ai_config.py:1]: Just revert whole file. This is hopefully in #590

File: scripts/main.py [hunk 3]
```diff
@@ -167,221 +132,16 @@ def construct_prompt():
...
             speak_text=True)
         should_continue = input(f"""Continue with the last settings? 
-Name:  {config.ai_name}
-Role:  {config.ai_role}
-Goals: {config.ai_goals}  
-Continue (y/n): """)
+    Name:  {config.ai_name}
+    Role:  {config.ai_role}
+    Goals: {config.ai_goals}  
+    Continue (y/n): """)
```
Comment (nponeccop) [inline scripts/main.py:138]: Revert the indentation change in an unrelated fragment

**Review by nponeccop — CHANGES_REQUESTED** [review]
Please ensure it only does one thing and doesn't do anything unrelated. The maintainer won't merge PRs silently adding or removing (this happens too!) features.

**Pull request closed without merging**